### ISSUE TITLE
:summon Kommando

### ISSUE THREAD

**Issue by joen** [HackerspaceBielefeld/HSBot2#6]
Es könnte ein Kommando eingeführt werden, das mehrere Benutzer informiert, wenn sie beschworen werden.
":summon admin" würde eine Direktnachricht (XMPP) an die als Admin hinterlegten JIDs senden: "Du wurdest um [Zeitpunkt der Beschwörung] von [beschwörende JID] beschworen"

andere Beschwörungen könnten sein:
- :summon keymaster (Alle, die einen Schlüssel haben / RFID-Schlüssel anlegen können)
- ...

**Comment by electronicfreak** [comment]
wer soll denn entscheiden wer dann gerufen wird?

**Comment by joen** [comment]
Ja, das wäre noch so eine Frage ... eine zentrale Stelle, wo steht, wer offizielle Posten innehat wäre ja auch mal gut. Evtl. im neuen LDAP?

**Comment by electronicfreak** [comment]
wie wärs denn mit :summon-add <group> und :summon-rem <group> und jeder darf das selbst entscheiden? auf gut glück das mal jemand ldap macht bau nix mehr

**Comment by electronicfreak** [comment]
mit einer private nachrichten bot geschichte könnte man da was machen. dann kann man ihm auch andere suchworte mitgeben die dir dann gemeldet werden. was meinste?

**Comment by electronicfreak** [comment]
besteht da noch interesse? möchtest dus evtl selbst mal versuchen? die funktion würde dann getriggert mit 2 parametern und zwar nick und dem gesendeten text

**Comment by joen** [comment]
ja, Interesse besteht noch. Ich würd's auch gerne selber probieren ... Wir sollten das nochmal besprechen.
